index.js

```diff
@@ -192,7 +192,7 @@ app.get('/:listIds/:mdbListKey/:userKey?/manifest.json', (req, res) => {
 						catalogs.push(catalogClone)
 					})
 					manifestClone.catalogs = catalogs
-					res.setHeader('Cache-Control', `public, max-age=${24 * 60 * 60}`)
```

Comment:
i think this should be reverted as it is just the manifest of the addon, it does not affect the catalog update time, the manifest should normally never change